collections/_article/godotcon-2023-cfp.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Godot Conference 2023: Call for Participations"
+excerpt: "GodotCon 2023 needs participations"
+categories: ["events"]
+author: Julian
+image: /storage/blog/covers/godotcon-2023.webp
+image_caption_description: Image by Raffaele Picca
+date: 2023-08-06 17:00:00
+---
+
+We are organizing the annual Godot Conference that will happen [live in Munich](/article/godotcon-2023/) on November 4th and 5th 2023 for the 
+first time since [the last occurrence in 2020](/article/meet-community-fosdem-and-godotcon-2020/). We are really thrilled to meet again fellow Godot users and developers to share experiences during the two days of the event.
```

Comment:
```suggestion
We are organizing the annual Godot Conference that will happen [live in Munich](/article/godotcon-2023/) on November 4th and 5th 2023 for the 
first time since [our last conference in 2020](/article/meet-community-fosdem-and-godotcon-2020/). We are really thrilled to meet fellow Godot users and developers again and to share experiences during the two days of the event.
```